nginx/src/css/chat.css

```diff
@@ -0,0 +1,226 @@
+#chat-container {
+	position: fixed;
+	bottom: 0;
+	right: 0;
+	width: 300px;
+	height: 500px;
+	border: none;
+	box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+	display: flex;
+	flex-direction: column;
+	border-radius: 8px;
+	transition: all 0.3s ease;
+	z-index: 1000;
+	text-align: center;
+	background-color: #d4d7d9;
+}
+
+#chat-container.toggled {
+	margin-bottom: -500px;
+}
+
+#chat-title {
+	padding: 5px;
+	background-color: #d4d7d9;
+	text-align: center;
+	font-weight: bold;
+	font-size: 23px;
+	color: white;
+	border-radius: 7px;
+	border: none;
+}
+
+#chat-log {
+	flex-grow: 1;
+	padding: 10px;
+	border: none;
+	overflow-y: auto;
+	display: flex;
+	flex-direction: column;
+	height: calc(100% - 100px);
+	background-image: url('../assets/chat-background2.webp');
```

Comment:
maybe add fallback background-color in case image doesn't load

Reply:
if the image does not load, the background will be the base grey, so it will be the same as the rest of the image